packages/perseus/src/widgets/interactive-graph.tsx

```diff
@@ -2600,19 +2602,20 @@ class InteractiveGraph extends React.Component<Props, State> {
             ) {
                 const guess = userInput.coords;
                 const correct = rubric.correct.coords;
+                const allowReflexAngles =
+                    rubric.correct.allowReflexAngles || false;
 
                 let match;
                 if (rubric.correct.match === "congruent") {
                     const angles = _.map([guess, correct], function (coords) {
-                        const angle = GraphUtils.findAngle(
-                            // @ts-expect-error - TS2532 - Object is possibly 'undefined'.
-                            coords[2],
-                            // @ts-expect-error - TS2532 - Object is possibly 'undefined'.
-                            coords[0],
-                            // @ts-expect-error - TS2532 - Object is possibly 'undefined'.
-                            coords[1],
+                        if (!coords) {
+                            return false;
+                        }
+                        const angle = getClockwiseAngle(
+                            coords,
+                            allowReflexAngles,
                         );
```

Comment:
While there aren't any graphs that currently make use of the reflexive angles, this update in validation logic is necessary to make sure that we can support them properly moving forward. 

I think it is safe to update the validator at this time as the Legacy Angle Graph is no longer being shown to users.